app/renderer/components/preferences/paymentsTab.js

```diff
@@ -244,6 +245,7 @@ class PaymentsTab extends ImmutableComponent {
             showOverlay={this.props.showOverlay}
             hideOverlay={this.props.hideOverlay}
             setOverlayName={this.props.setOverlayName}
+            inProgress={this.props.ledgerData.get('status') === ledgerStatuses.IN_PROGRESS}
```

Comment:
See above

Reply:
I can define it above but above this is preferences with a lot getting set in renderer state. I'd prefer to leave this one 'as is' (at least in this file).

Reply:
Done.